optuna/visualization/_timeline.py

```diff
@@ -107,28 +116,29 @@ def _get_timeline_info(study: Study) -> _TimelineInfo:
 
 def _get_timeline_plot(info: _TimelineInfo) -> "go.Figure":
     _cm = {
-        "COMPLETE": "blue",
+        "COMPLETE": "#cccccc",
         "FAIL": "red",
         "PRUNED": "orange",
         "RUNNING": "green",
         "WAITING": "gray",
     }
 
     fig = go.Figure()
-    for s in sorted(TrialState, key=lambda x: x.name):
-        bars = [b for b in info.bars if b.state == s]
+    for s, infeasible in product(sorted(TrialState, key=lambda x: x.name), [False, True]):
+        color = "#cccccc" if infeasible else _cm[s.name]
```

Comment:
@mamu suggested that only infeasible and complete trials will be gray, but the current implementation changes the color of all infeasible trials to gray, regardless of their state.